x/feedist/handler.go

```diff
@@ -145,22 +145,22 @@ func (h *distributeHandler) validate(ctx weave.Context, db weave.KVStore, tx wea
 	return msg, nil
 }
 
-type updateRevenueHandler struct {
+type resetRevenueHandler struct {
 	auth   x.Authenticator
 	bucket *RevenueBucket
 	ctrl   CashController
 }
 
-func (h *updateRevenueHandler) Check(ctx weave.Context, db weave.KVStore, tx weave.Tx) (weave.CheckResult, error) {
+func (h *resetRevenueHandler) Check(ctx weave.Context, db weave.KVStore, tx weave.Tx) (weave.CheckResult, error) {
 	var res weave.CheckResult
 	if _, err := h.validate(ctx, db, tx); err != nil {
 		return res, err
 	}
-	res.GasAllocated += updateRevenueCost
+	res.GasAllocated += resetRevenueCost
```

Comment:
Can you multiply this by the number of recipients?
I think that is a more realistic estimate of the workload